froide/foirequest/models/message.py

```diff
@@ -37,10 +37,13 @@ def get_throttle_filter(self, queryset, user, extra_filters=None):
             qs = qs.filter(**extra_filters)
         return qs, "timestamp"
 
+    def with_drafts(self, drafts=True):
```

Comment:
Added here since I can't add it to the manager class directly:

Since we have a seperate proxy type for drafts, I would opt for not returning the drafts in the default manager (and the default related manager if easily possible)

Reply:
That would require a seperate API endpoint for message drafts as well then. Since we're also splitting up the admin for drafts and non-drafts, I suppose it's okay to do that? It just feels a little repetitive to me, since they only differ in one attribute.

Reply:
As discussed today, let's add two endpoints